### ISSUE TITLE
Tenor sax, bari sax wrong fingerings

### ISSUE THREAD

**Issue by angusmoncrieff** [eduardomourar/fingering-diagram#19]
Hi there,

It seems the fingerings for tenor and bari are all offset - see screenshot attached: C major should have same fingerings for all the saxes, but tenor and bari are blank for first notes, and then the fingerings that do appear are wrong.

If I'm understanding the code correctly, I think this is the fix:

```
            if (instrument === 'wind.reed.saxophone.alto') {
                this.transpose = 7;
            } else if (instrument === 'wind.reed.saxophone.tenor') {
                this.transpose = 12;         // was 5 - tenor sounds an octave lower than sop
            } else if (instrument === 'wind.reed.saxophone.baritone') {
                this.transpose = 19;         // was 7
                this.base += '\uE281';
            }
```

Many thanks for your great plugin. FYI I'm 'borrowing' the logic/codes for a web app I'm building - hope that's ok? Will credit you.

Regards,
Angus

![Screenshot 2020-09-16 at 16 18 15](https://user-images.githubusercontent.com/47029910/93357488-448a6900-f838-11ea-8bad-7a21a3aff8f6.png)

**Comment by eduardomourar** [comment]
@gusmoney, thanks for reporting this bug and a fix. In regards to the code, feel free to use it and the Fiati font as well. I am fine just pointing to this repository. I would suggest adding the `fiati-regular.woff` to your web app, because it is a smaller version of the font. I have not really tried the woff one, so I would appreciate if you can give me your feedback here (or in [Fiati repo](https://github.com/eduardomourar/fiati) in case of a bug) .

**Comment by eduardomourar** [comment]
Released [v1.4.0](https://github.com/eduardomourar/fingering-diagram/releases/tag/v1.4.0) with this fix.
